src/metabase/email/messages.clj

```diff
@@ -324,10 +319,15 @@
     (email/send-message! message)))
 
 (defn- make-message-attachment [[content-id url]]
-  {:type         :inline
-   :content-id   content-id
-   :content-type "image/png"
-   :content      url})
+  (if (str/includes? content-id "_error")
```

Comment:
I'm open to any suggestions about how to better handle this.
Basically, `make-message-attachment` is built with the assumption that all attachments from the render code are images. So, when I passed an attachment that was a text file, it was at first being opened as if it were an image. This obviously doesn't work.

So, I did this, relying on the content_id string to indicate.

Reply:
I'd check for the url extension here.